test/form-functions.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once '../form-functions.php';
+use PHPUnit\Framework\TestCase;
+
+class FormFunctions extends TestCase {
+
+    public function testSuccessFormFunctions() {
```

Comment:
your missing quite a few test scenarios. you need to test every eventuality of your code. What if the rating is above 10? what if it is not an int? what if its below 1?